src/shared/components/chips/chip.css.ts

```diff
@@ -0,0 +1,106 @@
+import { style, styleVariants } from "@vanilla-extract/css";
+import { color } from "src/styles/tokens/color.css";
+import { fontStyles } from "src/styles/tokens/font-styles.css";
+
+const sizeWidths = {
+  small: "7.5rem",
+  medium: "12.2rem",
+  large: "14.8rem",
+  dropdown: "10.8rem",
+};
+
+const clipPathValue =
+  "polygon(92.243% 0%, 11.748% 0%, 11.748% 0%, 11.362% 0.027%, 10.979% 0.107%, 10.6% 0.241%, 10.227% 0.426%, 9.86% 0.662%, 9.501% 0.948%, 9.15% 1.284%, 8.81% 1.668%, 8.481% 2.1%, 8.163% 2.58%, 2.421% 11.863%, 2.421% 11.863%, 1.983% 12.643%, 1.584% 13.501%, 1.226% 14.43%, 0.911% 15.423%, 0.639% 16.473%, 0.413% 17.573%, 0.235% 18.716%, 0.106% 19.894%, 0.027% 21.101%, 0% 22.33%, 0% 83.148%, 0% 83.148%, 0.102% 85.881%, 0.395% 88.474%, 0.866% 90.892%, 1.497% 93.1%, 2.272% 95.064%, 3.176% 96.748%, 4.192% 98.119%, 5.305% 99.141%, 6.499% 99.779%, 7.757% 100%, 88.605% 100%, 88.605% 100%, 88.996% 99.972%, 89.383% 99.89%, 89.765% 99.754%, 90.142% 99.566%, 90.512% 99.325%, 90.874% 99.033%, 91.228% 98.691%, 91.571% 98.299%, 91.902% 97.858%, 92.222% 97.37%, 97.611% 88.541%, 97.611% 88.541%, 98.043% 87.76%, 98.437% 86.904%, 98.79% 85.977%, 99.102% 84.988%, 99.37% 83.943%, 99.592% 82.849%, 99.768% 81.714%, 99.896% 80.543%, 99.974% 79.344%, 100% 78.124%, 100% 16.852%, 100% 16.852%, 99.898% 14.119%, 99.605% 11.526%, 99.134% 9.108%, 98.503% 6.9%, 97.728% 4.936%, 96.824% 3.252%, 95.808% 1.881%, 94.695% 0.859%, 93.501% 0.221%, 92.243% 0%)";
+
+const baseCssProps = {
+  clipPath: clipPathValue,
+  boxSizing: "border-box" as const,
```

Comment:
as const 사용한 이유가뭐야?

Reply:
타입 에러가 왜 발생하고있는거야?? 타입 단언하는 건 되도록이면 지양해야하는데 

```suggestion
const style: { boxSizing: 'border-box' } = {
  boxSizing: "border-box",
};
```

이렇게도 할 수 있을 것 같고 스타일 객체를 근데 이렇게 선언해야 할 이유가 있는걸까?

Reply:
vanilla-extract 자체에서 `type BoxSizing = 'border-box' | 'content-box' | 'initial' | 'inherit';` 와 같이 boxSizing 속성에 대해 type을 정의해 둔다고 합니다, 근데 `boxSizing : "border-box"` 이렇게 작성하면  본인의 타입 중 하나라고 인식하지 않고 그냥 string 자체로 인식한다고 합니다. 그래서 typescript가 안전하지 않다고 판단해서 경고를 띄워주는데 타입 단언을 해주면서 `string`이 아닌 `"border-box"` 자체로 취급할 수 있게 되어 할당이 가능해지는 상태가 되어서 타입 단언을 사용하게 되었습니다!

Reply:
아티클 하나 나왔어.